_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: drupal drupal8 drupal-planet marketing 
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
+featured_image: "/assets/img/blog/supercharged-seo-with-drupal-8.jpg"
+featured_image_alt: "Supercharged SEO with Drupal 8"
+---
+
+Here at Savas Labs, we listen to our clients needs, and what many of our clients need is to reach their target audiences effectively. Let’s be honest here - perfectly coded, a pretty looking website will be of little use if it doesn’t produce leads / facilitate conversions or generate revenue! So how do we help our clients achieve their goals? We get there by balancing website objectives while giving priority to lead generation via SEO. The more quality traffic that comes to the website - the more conversions we can achieve. It’s that simple!
```

Comment:
I think this point may have been discussed already, but we also work on websites (like the news publishing one we just finished) that don't produce leads, conversions or generate revenue, but which are still of great use. So maybe consider adjusting this sentence?